src/state/slices/portfolioSlice/utils.ts

```diff
@@ -270,13 +227,51 @@ export const accountToPortfolio: AccountToPortfolio = ({
         break
       }
       default:
-        break
+        assertUnreachable(chainNamespace)
     }
   })
 
   return portfolio
 }
 
+export const checkAccountHasActivity = (account: Account<ChainId>) => {
```

Comment:
This helper means we can share logic when deriving account activity in account management, and also when adding accounts to redux via the portfolio api.